system/jlib/jsem.cpp

```diff
@@ -26,6 +26,21 @@
 #include <sys/time.h>
 #include <semaphore.h>
 
+
+void getEndTime(timespec & abs, unsigned timeout)
+{
+    timeval cur;
+    gettimeofday(&cur, NULL);
```

Comment:
why not clock_gettime(CLOCK_MONOTONIC[_RAW], ...) ?

Reply:
Not sure - this was moving a preexisting call into a common function.

Reply:
Opened HPCC-20018 to change it.